src/components/admin/agency/agency-profile-form.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles'; 
+import { Grid, TextField, Button, Box } from '@material-ui/core';
+import { Formik } from 'formik';
+import ImageOutlinedIcon from '@material-ui/icons/ImageOutlined';
+import LogoUpload from '../agency/logo-upload'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
```

Comment:
I don't think we need to make custom styles. Try to use what we have in Material UI